server/rest/src/main/java/org/infinispan/rest/resources/MetricsResource.java

```diff
@@ -42,6 +42,10 @@
    private final MetricsRequestHandler requestHandler = new MetricsRequestHandler();
 
    public MetricsResource() {
+      registerBaseMetrics();
+   }
+
+   private void registerBaseMetrics() {
```

Comment:
This method was extracted just in case we need to replace it for quarkus.

Reply:
shouldn't be protected?

Reply:
For quarkus? I don't know. Maybe @wburns ?

Reply:
@anistor I'm pretty sure Quarkus can replace a private method, so it doesn't need to be protected. But it would be nice to have a comment in the code, to prevent others from trying to inline it and finding out that we need it for Quarkus after the PR is integrated.

Reply:
done!